README.md

```diff
@@ -84,9 +84,7 @@ Below is the command to run the `redis-server` application with the ELF Loader.
 Note the rootfs contents were copied beforehand.
```

Comment:
I think this line should now be removed given that rootfs contents are no longer copied.

Reply:
I agree will update the PR